core/modules/date/views/date_views_filter_handler_simple.inc

```diff
@@ -283,9 +283,10 @@ class date_views_filter_handler_simple extends views_handler_filter_date {
    */
   function value_form(&$form, &$form_state) {
     // We use different values than the parent form, so we must
-    // construct our own form element.
+    // Construct our own form element.
```

Comment:
Why this??

Reply:
Coding standards (Start with uppercase)

Reply:
@jenlampton that's in the *middle of a sentence*. :smile: " so we must Construct"